WL-4160 Changed the colour of the login table for 'xlogin.vm' file.

ouit0408 opened the pull request.

buckett commented on library/src/webapp/skin/neo-ox/tool.css:
```diff
@@ -1019,9 +1019,10 @@ tr.selectedFound td {
     font-weight: normal;
 }
 .login {
-    border: 1px solid #dedede;
```
Isn't this used in other places than just the login page?

ouit0408 replied:
It is used at two other places.I will make changes in the file and not in this css.

ouit0408 commented:
Since the class .login was used at other places too , I have added new class '.oulogin' to the table in xlogin.vm file. Added colour to this new class 'oulogin' for the table. 
New pull request https://github.com/ox-it/wl-login/pull/6 . sorry about this my fault i didn't check the references for the class.

ouit0408 committed:
WL-4160 Changed the colour of the login table for 'xlogin.vm' file by
adding new css class 'oulogin'.

buckett commented:
Closing as we're not doing the move to 960px.
